bin/polly.py

```diff
@@ -249,12 +249,12 @@ def PositiveInt(string):
   build_tag = polly_toolchain
 
 """Tune environment"""
-if toolchain_entry.name == 'mingw' or toolchain_entry.name == 'mingw-c11':
+if toolchain_entry.name == 'mingw' or toolchain_entry.name == 'mingw-c11' or toolchain_entry.name == 'mingw-cxx14' or toolchain_entry.name == 'mingw-cxx17':
   mingw_path = os.getenv("MINGW_PATH")
   detail.verify_mingw_path.verify(mingw_path)
   os.environ['PATH'] = "{};{}".format(mingw_path, os.getenv('PATH'))
 
-if toolchain_entry.name == 'msys':
+if toolchain_entry.name == 'msys' or toolchain_entry.name == 'msys-cxx14' or toolchain_entry.name == 'msys-cxx17':
```

Comment:
Same here, use `startswith('msys')`.

Reply:
Will do.  Python is not my strong suit.  Thanks.

Reply:
corrected.